packages/dscc-gen/src/config.ts

```diff
@@ -16,7 +16,7 @@ import {assertNever} from './util';
 import * as util from './util';
 import {
   addBucketPrefix,
-  checkGsutilInstalled,
+  //  checkGsutilInstalled,
```

Comment:
remove instead of comment

Reply:
forgot this was commented out. I think I actually want to keep this check.